source/backend/api/Models/Concepts/CompensationRequisition/CompensationPayeeMap.cs

```diff
@@ -0,0 +1,41 @@
+using Mapster;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Models.Concepts
+{
+    public class CompensationPayeeMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.PimsAcquisitionPayee, CompensationPayeeModel>()
+                .PreserveReference(true)
+                .Map(dest => dest.AcquisitionPayeeId, src => src.AcquisitionPayeeId)
+                .Map(dest => dest.CompensationRequisitionId, src => src.CompensationRequisitionId)
+                .Map(dest => dest.AcquisitionOwnerId, src => src.AcquisitionOwnerId)
+                .Map(dest => dest.InterestHolderId, src => src.InterestHolderId)
+                .Map(dest => dest.OwnerRepresentativeId, src => src.OwnerRepresentativeId)
+                .Map(dest => dest.OwnerSolicitorId, src => src.OwnerSolicitorId)
+                .Map(dest => dest.MotiSolicitorId, src => src.AcquisitionFilePersonId)
```

Comment:
Hmm, I guess how this works is that the payee only references acquisitionfilePersonId values with type Moti_Solicitor?

Reply:
Correct, that is validated on the front end. The backend only knows that is part of the acq team.